server/pkg/asset/preview_type.go

```diff
@@ -9,41 +9,29 @@ import (
 type PreviewType string
 
 const (
-	PreviewTypeIMAGE   PreviewType = "IMAGE"
-	PreviewTypeGEO     PreviewType = "GEO"
-	PreviewTypeGEO3D   PreviewType = "GEO3D"
-	PreviewTypeMODEL3D PreviewType = "MODEL3D"
+	PreviewTypeImage   PreviewType = "image"
+	PreviewTypeGeo     PreviewType = "geo"
+	PreviewTypeGeo3d   PreviewType = "geo3d"
+	PreviewTypeModel3d PreviewType = "model3d"
+	PreviewTypeUnknown PreviewType = "unknown"
 )
 
-func PreviewTypeFrom(p string) (PreviewType, bool) {
-	switch PreviewType(p) {
-	case PreviewTypeIMAGE:
-		return PreviewTypeIMAGE, true
-	case PreviewTypeGEO:
-		return PreviewTypeGEO, true
-	case PreviewTypeGEO3D:
-		return PreviewTypeGEO3D, true
-	case PreviewTypeMODEL3D:
-		return PreviewTypeMODEL3D, true
-	default:
-		return PreviewType(""), false
-	}
-}
-
-func PreviewTypeFromRef(p *string) *PreviewType {
+func PreviewTypeFrom(p *string) *PreviewType {
```

Comment:
I think it is good that `PreviewTypeFrom` and `PreviewTypeFromRef` are available since `lo.ToPtr` can be reduced.

Reply:
`PreviewTypeFrom` is not used anywhere that is why I deleted it, should I rename PreviewTypeFrom to PreviewTypeFromRef?

Reply:
I saw `PreviewTypeFrom(lo.ToPtr(hogehoge))` at https://github.com/reearth/reearth-cms/pull/320/files#diff-5973a8c58a7fed17669414bc79a16b0bd282de7f6cf6050d7333b88f73230c67R372, for example.

Reply:
Done!